packages/generator-electrode/generators/app/templates/gulpfile.js

```diff
@@ -1 +1,2 @@
+process.env.ES6_LINT = true;
```

Comment:
`process.env.SERVER_ES6`

Doesn't the archetype need to look at this flag and decide to use ES6 eslint to lint the code in `src/server`?